content/news/2021-04-04-bevy-0.5/index.md

```diff
@@ -0,0 +1,1254 @@
+commands.spawn_bundle(SpriteBundle {
+    material: material.clone(),
+    transform: Transform::from_xyz(150.0, 0.0, 0.0),
+    ..Default::default()
+});
+commands.spawn_bundle(SpriteBundle {
+    material,
+    transform: Transform::from_xyz(-150.0, 0.0, 0.0),
+    sprite: Sprite {
+        // Flip the logo to the left
+        flip_x: true,
+        // And don't flip it upside-down ( the default )
+        flip_y: false,
+        ..Default::default()
+    },
+    ..Default::default()
+});
+```
+
+## Color Spaces
+
+<div class="release-feature-authors">authors: @mockersf</div>
+
+Color is now internally represented as an enum, which enables lossless (and correct) color representation. This is a significant improvement over the previous implementation, which internally converted all colors to linear sRGB (which could cause precision issues). Colors are now only converted to linear sRGB when they are sent to the GPU. We also took this opportunity to fix some incorrect color constants defined in the wrong color space.
+
+```rust
+pub enum Color {
+    /// sRGBA color
+    Rgba {
+        /// Red component. [0.0, 1.0]
+        red: f32,
+        /// Green component. [0.0, 1.0]
+        green: f32,
+        /// Blue component. [0.0, 1.0]
+        blue: f32,
+        /// Alpha component. [0.0, 1.0]
+        alpha: f32,
+    },
+    /// RGBA color in the Linear sRGB colorspace (often colloquially referred to as "linear", "RGB", or "linear RGB").
+    RgbaLinear {
+        /// Red component. [0.0, 1.0]
+        red: f32,
+        /// Green component. [0.0, 1.0]
+        green: f32,
+        /// Blue component. [0.0, 1.0]
+        blue: f32,
+        /// Alpha component. [0.0, 1.0]
+        alpha: f32,
+    },
+    /// HSL (hue, saturation, lightness) color with an alpha channel
+    Hsla {
+        /// Hue component. [0.0, 360.0]
+        hue: f32,
+        /// Saturation component. [0.0, 1.0]
+        saturation: f32,
+        /// Lightness component. [0.0, 1.0]
+        lightness: f32,
+        /// Alpha component. [0.0, 1.0]
+        alpha: f32,
+    },
+}
+```
+
+## Wireframes
+
+<div class="release-feature-authors">authors: @Neo-Zhixing</div>
+
+Bevy can now draw wireframes using the opt-in `WireframePlugin`
+
+![wireframe](wireframe.png)
+
+These can either be enabled globally or per-entity by adding the new `Wireframe` component.
+
+## Simple 3D Game Example: Alien Cake Addict
+
+<div class="release-feature-authors">authors: @mockersf</div>
+
+This example serves as a quick introduction to building 3D games in Bevy. It shows how to spawn scenes, respond to input, implement game logic, and handle state transitions. Pick up as many cakes as you can!
+
+![alien_cake_addict](alien_cake_addict.png)
+
+## Timer Improvements
+
+<div class="release-feature-authors">authors: @kokounet</div>
+
+Timers now internally use `Duration`s instead of using `f32` representations of seconds. This both increases precision and makes the api a bit nicer to look at.
```

Comment:
what is `Duration`? is this part of std or a bevy type? a link here to the docs would be nice!

Reply:
I see it is part of std now - maybe just a small adjustment here for clarity

```suggestion
Timers now internally use `std::time::Duration`s instead of using `f32` representations of seconds. This both increases precision and makes the api a bit nicer to look at.
```

Reply:
Maybe it could link to the official docs? https://doc.rust-lang.org/std/time/struct.Duration.html

Reply:
I'm planning on doing a pass that will make all of the "types" link to docs (like we've done in past blog posts). I'll do that here.